xesmf/backend.py

```diff
@@ -277,7 +278,8 @@ def esmf_regrid_build(sourcegrid, destgrid, method,
     regrid = ESMF.Regrid(sourcefield, destfield, filename=filename,
                          regrid_method=esmf_regrid_method,
                          unmapped_action=ESMF.UnmappedAction.IGNORE,
-                         ignore_degenerate=ignore_degenerate)
+                         ignore_degenerate=ignore_degenerate,
+                         factors=True)
```

Comment:
I don't see where factors is used

Reply:
It tells ESMPy to store weights in memory instead of on disk. The rest happens in frontend.py, where the weights are retrieved in `Regridder._compute_weights`